stdlib/internal/src/main/ballerina/internal/jwt_common.bal

```diff
@@ -36,7 +36,7 @@ const string IAT = "iat";
 # + kid - Key ID, hint indicating which key was used to secure the JWS
 # + customClaims - Map of custom claims
 public type JwtHeader record {
-    string alg;
+    string alg = "";
```

Comment:
Don't we need to have some pre-define algorithm here, instead of empty string? Constant would be ideal.

Reply:
Yes. Created an issue[1] to track the requirement. Plan is to address this during the Auth refactoring.
[1] https://github.com/ballerina-platform/ballerina-lang/issues/11528